### PR TITLE
Setup bookstore app

### PR DESCRIPTION
**Pull request by GintsM**
In this PR 
 - React app
 - React Redux npm package
 - Structure feature folder approach
 - Reusable components
 - React Router -default `Books` second `Categories`
 - Form to add new book and button to remove.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const categories = () => (
+  <div>
+    <p>Under construction</p>
+  </div>
+);
+
+export default categories;
```
Comment (TedLivist) [inline src/redux/categories/categories.js:7]: - [ ] You created your Category component under the `redux` directory. All components should live under the `components` directory. This `redux` directory is solely meant for redux configurations. Kindly extract this component into the **components** folder.

File: src/redux/books/book.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+    <div className="current">
+      <h3>Current Chapter</h3>
+      <h3><b>Chapter 17</b></h3>
+      <input type="button" value="Update Progress" />
+    </div>
+  </div>
+);
+
+export default book;
```
Comment (TedLivist) [inline src/redux/books/book.js:27]: - [ ] Kindly extract this **Book** component into the **components** folder.

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import Book from './book';
+
+const books = () => (
+  <div>
+    <Book />
+    <AddBook />
+  </div>
+);
+
+export default books;
```
Comment (TedLivist) [inline src/redux/books/books.js:11]: - [ ] Kindly extract this **Books** component into the **components** directory.

File: src/components/Button.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+// Button.propTypes = {
+//   className: PropTypes.string,
+//   value: PropTypes.string,
+//   onEvent: PropTypes.func.isRequired,
+// };
+
+// Button.defaultProps = {
+//   className: '',
+//   value: '',
+// };
```
Comment (TedLivist) [inline src/components/Button.js:18]: - [ ] Let's keep our code clean. Kindly remove commented code.

File: src/redux/books/book.js (hunk 2 above)
Comment (TedLivist) [inline src/redux/books/book.js:27]: - [ ] Your **Book** component should take in the book data as a prop and render the title and author

File: src/components/Button.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// import PropTypes from 'prop-types';
```
Comment (TedLivist) [inline src/components/Button.js:1]: - [ ] Kindly remove commented code

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @GintsM,

Good job so far!
✔️ Nice UI
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] **Your README file is a generic file. Kindly update it to be more descriptive of the project you just built. I'd recommend using the Microverse template [here](https://github.com/microverseinc/readme-template)**

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 1 above)
Comment (TedLivist) [inline src/redux/categories/categories.js:7]: - [ ] Also extract this **Categories** component into the **components** folder

File: src/components/book.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+    <div className="current">
+      <h3>Current Chapter</h3>
+      <h3><b>Chapter 17</b></h3>
+      <input type="button" value="Update Progress" />
+    </div>
+  </div>
+);
+
+export default Book;
```
Comment (ArthurGC) [inline src/components/book.js:27]: - [ ] As the previous said, The `Book` component should take in the book data as a prop and render the title and author. A good source of info is [Components and Props](https://es.reactjs.org/docs/components-and-props.html).
- [ ] Don't forget this from the previous reviewer too. 😉 

review-readme

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @GintsM 

Great work on making the changes requested by a previous reviewer. 👏 
You've done well implementing some of the requested changes but there are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
Great job following most of the first reviewer's suggestions :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 7]
```diff
@@ -1,70 +1,14 @@
...
+## Will be Built With
 
-## Available Scripts
+- REACT REDUX
+- CSS
 
-In the project directory, you can run:
+## Author
+  * GitHub: [GintsM](https://github.com/GintsM) 
+  * LinkedIn: [Gints Misins](https://www.linkedin.com/in/gints-misins-756b2321a/)
```
[the remark this answers is not in the document]
Comment (GintsM) [reply README.md:11]: What is professional?  Blank Screenshot from a template, or description of what at this moment is unknown, Question is - Do I provide You with enough information on what so far is done? Can You please answer this question? If no please add what is missing.

[the remark this answers is not in the document]
Comment (mke2111) [reply README.md:11]: A professional readme file contains a project name, a short project description, technologies used, and how to get started on a computer.
You definitely don't have to provide a blank screenshot for this case. Eliminate this for now. You can always add one once you have something set up and complete.

**Pull request merged**